.circleci/config.yml

```diff
@@ -716,6 +744,17 @@ workflows:
                 - /^testflow[\d|.]+$/
                 - /^premainnet-[\d|.]+$/
                 - /^v[\d|.]+-release$/
+      - build-individual-crates:
+          requires:
+            - prefetch-crates
+          filters:
+            branches:
+              ignore:
+                - gh-pages
```

Comment:
unfortunately this "gh-pages" doesn't matter.   Since this file will never be in gh-pages branch the config will never be honored.   The one time I tried putting a minimal ignore-this-branch circle ci config in gh-pages it was immediately overwritten.   So line 753 will do no good.  It will also do no harm aside from cause a little confusion.